### PR TITLE
Plot CRUD commands for admins

### PR DESCRIPTION
**Pull request by claresloggett**
Add commands to manage plots within projects.

To be useable need to deploy server PR https://github.com/AusTrakka/austrakka-portal/pull/901

### CODE REVIEW

File: austrakka/utils/helpers/project.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from austrakka.utils.api import api_get
+from austrakka.utils.paths import PROJECT_PATH
+
+
+def get_project_by_abbrev(abbrev: str):
+    response = api_get(
```
Comment (samcarswell) [inline austrakka/utils/helpers/project.py:6]: You can now do this in one line with `return api_get(path=f"{PROJECT_PATH}/abbrev/{abbrev}")['data']`

`api_get` throws an `UnknownResponseException` if data is not included so there's no need to check or conditionally return the entire response as all endpoints now return the standard response

File: austrakka/components/plot/funcs.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+def list_plot_types():
+    response = api_get(
+        path=f'{PLOT_PATH}/plottypes'
+    )
+    for plottype in response['data']:
+        # pylint: disable=print-function
+        print(plottype['name'])
+
+
+def _get_plot_by_abbrev(abbrev: str):
```
Comment (samcarswell) [inline austrakka/components/plot/funcs.py:124]: Can we move this to `utils/helpers` to be consistent

**Review by samcarswell — CHANGES_REQUESTED** [review]

**Comment by samcarswell** [comment]
Don't merge this until AB#4121 has been merged to dev and the options here fixed

**Comment by claresloggett** [comment]
@samcarswell I think you've already resolved the outstanding comments you had yourself - was there anything else?

**Review by samcarswell — APPROVED** [review]

**Pull request merged**